### DISCUSSION TITLE
Can multicast be used in ZUI without HA?

### DISCUSSION THREAD

**Discussion by apella12** [zwave-js/zwave-js-ui#4074] [Q&A] [answered]
Looking at multicast Mqtt set commandto be executed in ZUI. Not a HA user. Most documentation seems related to HA.  Tried a couple of things,  based on this;

@robertsLando So I've tested this around a dozen times on a variety of devices and it seems to work pretty flawlessly. The weirdness I had before with one switch that didn't go off the full way has resolved itself. Some of my switches do in fact call back in their new state, so they update in HA. For others, they do not change as we're still waiting on @AlCalzone for https://github.com/zwave-js/node-zwave-js/issues/1550. I think this is ready to go.

```
Topic: homeassistant/_CLIENTS/ZWAVE_GATEWAY-MQTT/multicast/set

{
"commandClass": 32,
"property": "targetValue",
"value": 255,
"nodes": [3, 5, 9, 13, 14, 19, 20, 21, 70, 112, 113, 114, 116, 117, 125]
}

```
but they just publish and nothing happens
```
zwave1
_CLIENTS
ZWAVE_GATEWAY-zwave-js-ui
version = {"value":"9.27.8","time":1734816592369}
status = {"value":true,"time":1734816592369}
multicast = {"args":[37,"targetValue",true,[4,5]]}
set = {"args":[37,"targetValue",true,[4,5]]}
api
refreshCCValues = {"success":true,"message":"Success zwave api call","args":[62,66],"origin":{"args":[62,66]}}
set = {"args":[62,66]}
multicast = {"success":false,"message":"Unknown API","args":[37,"targetValue",true,[4,5]],"origin":{"args":[37,"targetValue",true,[4,5]]}}
set = {"args":[37,"targetValue",true,[4,5]]}
```

**Comment by AlCalzone** [comment]
I don't know what the logs at the bottom are, but there seems to be an error:
```
multicast = {"success":false,"message":"Unknown API"
```

**Comment by robertsLando** [comment]
> Can multicast be used in ZUI without HA? 

I don't understand the question as the command you sent me (that is using mqtt apis) looks good.

**Comment by apella12** [comment, marked as answer]
Welcome back from holiday.

Comparing the first working `refreshCCValues` with the non-working `multicast` I assumed the `Unknown API` referred to the term `multicast` so assumed that was not an API command outside of HA.  I reordered the published args based on the docs for another try, but still not working
![unknownapi2025-01-07 090116](https://github.com/user-attachments/assets/5d7ebe4f-d5c1-465d-b2cf-6909c972946e)

 Do you see anything wrong or missing?


Edit: OK got it working.  Wrote it out longhand exactly like the docs
![multicast 2025-01-07 094313](https://github.com/user-attachments/assets/47ef5f81-1897-45c1-bee7-c03c7021a18e)

**Reply by robertsLando** [reply]
👍🏼
